src/components/CenteredBox.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface CenteredBoxProps {
+  children: React.ReactNode;
+}
+
+const CenteredBox: React.FC<CenteredBoxProps> = ({ children }) => {
+  return (
+    <div className="flex flex-col items-center justify-center mx-auto mt-40">
+      {children}
+    </div>
+  );
+};
+
+export default CenteredBox;
```

Comment:
[nits]
named exportするのが一般的かと思います

```suggestion
export const CenteredBox: React.FC<CenteredBoxProps> = ({ children }) => {
  return (
    <div className="flex flex-col items-center justify-center mx-auto mt-40">
      {children}
    </div>
  );
};
```